<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecognizeForm</class>
 <widget class="QDialog" name="RecognizeForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RecognizeForm</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;宋体&quot;;</string>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab1">
         <attribute name="title">
          <string>专用设备</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="verticalSpacing">
           <number>20</number>
          </property>
          <item row="0" column="1" colspan="3">
           <widget class="QGroupBox" name="groupBox_3">
            <property name="font">
             <font>
              <family>宋体</family>
              <pointsize>14</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="title">
             <string>图像DPI</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="0" column="0">
              <widget class="QComboBox" name="comboBox_ImageDPI">
               <property name="sizeAdjustPolicy">
                <enum>QComboBox::AdjustToContents</enum>
               </property>
               <item>
                <property name="text">
                 <string>200</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>300</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>600</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QPushButton" name="Button_ScannerControl1">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>50</height>
             </size>
            </property>
            <property name="text">
             <string>开始扫苗</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QPushButton" name="pushButton">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>50</height>
             </size>
            </property>
            <property name="text">
             <string>打开阅读仪</string>
            </property>
           </widget>
          </item>
          <item row="6" column="2">
           <widget class="QLabel" name="Label_Pages1">
            <property name="text">
             <string>999999</string>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QPushButton" name="pushButton_2">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>50</height>
             </size>
            </property>
            <property name="text">
             <string>关闭阅读仪</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLabel" name="Label_Pages1Title">
            <property name="text">
             <string>阅读图像计数：</string>
            </property>
           </widget>
          </item>
          <item row="5" column="3">
           <widget class="QPushButton" name="pushButton_3">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>50</height>
             </size>
            </property>
            <property name="text">
             <string>停止扫苗</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="3">
           <widget class="QGroupBox" name="groupBox_2">
            <property name="font">
             <font>
              <family>宋体</family>
              <pointsize>14</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="title">
             <string>图像类型</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="2" column="1">
              <widget class="QRadioButton" name="radioButton_LightRed">
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="text">
                <string>红色光源</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QRadioButton" name="radioButton_LightBlue">
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="text">
                <string>蓝色光源</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0" rowspan="3">
              <widget class="QRadioButton" name="radioButton_TypeColor">
               <property name="text">
                <string>彩色</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0" rowspan="2">
              <widget class="QRadioButton" name="radioButton_TypeGray">
               <property name="text">
                <string>灰度</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QRadioButton" name="radioButton_LightGreen">
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="text">
                <string>绿色光源</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QRadioButton" name="radioButton_LightWhite">
               <property name="font">
                <font>
                 <family>宋体</family>
                 <pointsize>14</pointsize>
                 <weight>50</weight>
                 <italic>false</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="text">
                <string>白色光源</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="2" column="1" colspan="3">
           <widget class="QGroupBox" name="groupBox_4">
            <property name="title">
             <string>其他功能</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QCheckBox" name="CheckBox_IsRectify">
               <property name="text">
                <string>启用裁剪</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBox_DoubleCheck">
               <property name="font">
                <font>
                 <family>宋体</family>
                 <pointsize>14</pointsize>
                 <weight>50</weight>
                 <italic>false</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="text">
                <string>开启重张检测</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab2">
         <attribute name="title">
          <string>图像文件</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="1">
           <spacer name="verticalSpacer_5">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="Button_OpenDirs">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>60</height>
             </size>
            </property>
            <property name="text">
             <string>从文件夹导入图片文件</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="Button_OpenFiles">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>60</height>
             </size>
            </property>
            <property name="text">
             <string>从计算机导入图像文件</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" rowspan="2">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="2" rowspan="2">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="maximumSize">
         <size>
          <width>224</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;宋体&quot;;</string>
        </property>
        <property name="title">
         <string>识别</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="verticalSpacing">
          <number>20</number>
         </property>
         <item row="3" column="1">
          <widget class="QLabel" name="Label_ReadCount">
           <property name="styleSheet">
            <string notr="true">font:20pt &quot;宋体&quot;;</string>
           </property>
           <property name="text">
            <string>XXXXXX</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0" colspan="2">
          <widget class="QCheckBox" name="CheckBox_Rotate90">
           <property name="text">
            <string>被识别图像旋转90度</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>已有图像数：</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0" colspan="2">
          <widget class="QPushButton" name="Button_RecognizeControl">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>80</height>
            </size>
           </property>
           <property name="text">
            <string>启动识别</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="Label_TotalCount">
           <property name="styleSheet">
            <string notr="true">font:20pt &quot;宋体&quot;;</string>
           </property>
           <property name="text">
            <string>XXXXXX</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="TextBox_BinarizeThreshold">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>识别图像数：</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0" colspan="2">
          <widget class="QCheckBox" name="CheckBox_AutoFilterWhitePage">
           <property name="text">
            <string>自动过滤白图像</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <spacer name="verticalSpacer_6">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>二值化阈值：</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
